install_config/aggregate_logging.adoc

```diff
@@ -230,40 +239,28 @@ When setting the `*ENABLE_OPS_CLUSTER*` parameter to `*true*`, Fluentd splits
 logs between the Elasticsearch cluster and a cluster reserved for operations
 logs. This means a second Elasticsearch and Kibana are deployed. The deployments
 are distinguishable by the *-ops* included in their names.
-
-. Next, as a cluster administrator, deploy the template that is created by
-the deployer:
 +
-====
-----
-$ oc process logging-support-template | oc create -f -
-----
-====
-
-. Enable the Fluentd service account by editing the security context:
+This creates a deployer pod and prints its name. Wait until the pod
```

Comment:
Adding newer instructions regarding what to look for when running the deployer. This can be confusing when starting out.